config.toml

```diff
@@ -40,86 +40,27 @@ identifier = "community"
 name = "Community"
 weight = 4
 
-[[menu.reference]]
-identifier = "api"
-name = "Commands & API"
-weight = 4
-[[menu.reference]]
-parent = "api"
-name = "JS & Go Libraries"
-url = "https://github.com/ipfs/interface-ipfs-core/tree/master/SPEC"
-weight = 30
 
 [[menu.reference]]
 identifier = "go_implementation"
 name = "Go Implementation"
 weight = 5
-[[menu.reference]]
-parent = "go_implementation"
-name = "go-ipfs Core API"
-url = "/go/pkg/go-ipfs/core/coreapi"
-weight = 10
-[[menu.reference]]
-parent = "go_implementation"
-name = "go-ipfs Coreiface API"
-url = "/go/pkg/go-ipfs/core/coreapi/interface"
-weight = 11
-[[menu.reference]]
-parent = "go_implementation"
-name = "go-ipfs Core options API"
-url = "/go/pkg/go-ipfs/core/coreapi/interface/options"
-weight = 12
-[[menu.reference]]
-parent = "go_implementation"
-name = "go-ipfs-api Reference"
-url = "/go/pkg/go-ipfs-api"
-weight = 30
+
 
 [[menu.reference]]
 identifier = "js_implementation"
 name = "JS Implementation"
 weight = 5
-[[menu.reference]]
-parent = "js_implementation"
-name = "js-ipfs"
-url = "https://github.com/ipfs/js-ipfs"
-weight = 10
-[[menu.reference]]
-parent = "js_implementation"
-name = "js-ipfs-api"
-url = "https://github.com/ipfs/js-ipfs-api"
-weight = 20
 
 [[menu.reference]]
 identifier = "specs"
 name = "Specifications & Planning"
 weight = 6
 [[menu.reference]]
 parent = "specs"
-name = "IPFS Whitepaper"
-url = "https://github.com/ipfs/ipfs/blob/master/papers/ipfs-cap2pfs/ipfs-p2p-file-system.pdf"
-weight = 10
-[[menu.reference]]
-parent = "specs"
-name = "IPFS Specifications"
-url = "https://github.com/ipfs/specs"
+name = "Libp2p Specifications"
+url = "https://github.com/libp2p/specs"
 weight = 20
-[[menu.reference]]
-parent = "specs"
-name = "Roadmap to 1.0.0"
-url = "https://github.com/ipfs/ipfs/blob/master/ROADMAP-TO-1.0.0.md"
-weight = 30
-[[menu.reference]]
-parent = "specs"
-name = "Implementation Status"
-url = "https://github.com/ipfs/ipfs/blob/master/IMPLEMENTATION_STATUS.md"
-weight = 40
-[[menu.reference]]
-parent = "specs"
-name = "RFC List"
-title = "“Requests for Changes” (RFC) are proposals for major modifications to protocols, library interfaces, and community processes."
-url = "https://github.com/ipfs/rfcs"
-weight = 50
 
 [[menu.guides]]
```

Comment:
I'm hopelessly confused about how menu generation works in this system, but I would just point out that "Concepts" is a clickable link (going to a blank placeholder page) but "Guides" is unclickable.  Low priority, but it feels slightly inconsistent.  (I recognize you're just trying to get something out the door.)

See screenshot below.

Reply:
![image](https://user-images.githubusercontent.com/225111/53034462-0d6b9000-3428-11e9-9e05-3630155579cf.png)